src/Native/System.Native/pal_process.cpp

```diff
@@ -389,12 +389,20 @@ extern "C" int32_t GetPriority(PriorityWhich which, int32_t who)
 {
     // GetPriority uses errno 0 to show succes to make sure we don't have a stale value
     errno = 0;
+#ifdef PRIORITY_REQUIRES_INT_WHO
+    return getpriority(which, who);
+#else
     return getpriority(which, static_cast<id_t>(who));
+#endif
 }
 
 extern "C" int32_t SetPriority(PriorityWhich which, int32_t who, int32_t nice)
 {
+#ifdef PRIORITY_REQUIRES_INT_WHO
```

Comment:
The build is failing on Linux and Mac. This needs to be #if not #ifdef since we're using #cmakedefine01.